commands/namespace.go

```diff
@@ -86,25 +86,28 @@ var namespaceGetCmd = &cobra.Command{
             }
         }
 
-        namespace, _, err := Client.Namespaces.Get(qualifiedName.GetNamespace())
-
-        if err != nil {
-            whisk.Debug(whisk.DbgError, "Client.Namespaces.Get(%s) error: %s\n", getClientNamespace(), err)
+        errHandler := func(err error, kind string) error {
+            whisk.Debug(whisk.DbgError, "Client.%s.List(%s) error: %s\n", getClientNamespace(), kind, err)
```

Comment:
the first `%s` in the `Client.%s.List(%s)` should be the entity name (i.e. Actions, Packages, etc) that matches the actual Client function name - not the namespace argument.

Reply:
Thanks - order fixed.